src/createServer.js

```diff
@@ -1,8 +1,32 @@
 'use strict';
 
-const createServer = () => {
-  // your code goes here
-};
+const express = require('express');
+const cors = require('cors');
+const { userRouter } = require('./routes/users.route');
+const { expensesRouter } = require('./routes/expenses.route');
+const { errorMiddleware } = require('./middlewares/errorMiddleware');
+
+function createServer() {
+  const app = express();
+
+  /* CORS  */
+  const options = {
+    origin: '*',
```

Comment:
Using a wildcard (`'*'`) for the `origin` in CORS configuration is not recommended for production environments as it allows requests from any domain. Consider specifying allowed origins or using environment variables to manage them.